### ISSUE TITLE
Remove now redundant rockstor-fstrim service and timer

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockstor-core#2442]
Our current os base in v4 releases are "Built on openSUSE" and as such now have what we previously supplemented in our older CentOS base: a built-in fstrim configuration:
```
Dec 12 12:43:38 rleap15-4 fstrim[27461]: /boot/efi: 30 MiB (31453184 bytes) trimmed on /dev/sda2
Dec 12 12:43:38 rleap15-4 fstrim[27461]: /: 20.6 GiB (22067851264 bytes) trimmed on /dev/sda3
Dec 12 12:43:38 rleap15-4 systemd[1]: fstrim.service: Deactivated successfully.
Dec 12 12:43:38 rleap15-4 systemd[1]: Finished Discard unused blocks on filesystems from /etc/fstab.
```
It is proposed that we no longer attempt to enable our own variant of this via our initrock.py (rockstor-pre.service) in-turn calling:

> scripts/flash-optimise.py

Which established the following systemd service and timer:
```
rockstor-fstrim.service
rockstor-fstrim.timer
```

**Comment by phillxnet** [comment]
Before the proposed changes we have:
```
rleap15-4:/opt/rockstor # /root/.local/bin/poetry run flash-optimize
2023-01-09 15:56:17,844: TRIM not supported on /dev/sda
2023-01-09 15:56:17,852: usb flash drive validation from udevadm: False
2023-01-09 15:56:17,852: flashiness of the drive(sda): False
```
After the proposed change (to be submitted shortly via pull request) we have:
```
rleap15-4:/opt/rockstor # /root/.local/bin/poetry run flash-optimize
2023-01-09 15:57:25,249: usb flash drive validation from udevadm: False
2023-01-09 15:57:25,249: flashiness of the drive(sda): False
```

**Comment by phillxnet** [comment]
Closing as:
Fixed by #2461
